src/AnsiUtils.ps1

```diff
@@ -0,0 +1,36 @@
+# Color codes from https://msdn.microsoft.com/en-us/library/windows/desktop/mt638032(v=vs.85).aspx
```

Comment:
Hehe, that is exactly the filename I'm using in my classes prototype branch.  Did some work on that last night but not quite ready yet for evaluation.